src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 101;
+    private final Random randomBallNumber = new Random();

```

Comment:
It seems to me, it is enough to name the variable just like 'random'

Reply:
Ok, maybe I overdid it a little in the "informativeness of variable names" )))